resqpy/organize/organize_old.py

```diff
@@ -82,7 +81,7 @@ def create_xml_has_occurred_during(model, parent_node, hod_pair, tag = 'HasOccur
     model.create_ref_node('ChronoTop', model.title_for_root(chrono_top_root), top_chrono_uuid, root = hod_node)
 
 
-def _alias_for_attribute(attribute_name):
+def alias_for_attribute(attribute_name):
```

Comment:
Should this be part of the public API? It is surely an implementation detail for attributes, hard to see why this should be exposed

Reply:
This method is already being called from other modules (e.g. well.py), therefore the renaming is correct.